src/app/services/technical-record/technical-record.service.ts

```diff
@@ -64,7 +92,38 @@ export class TechnicalRecordService {
     return this.http.get<VehicleTechRecordModel[]>(url, { responseType: 'json' });
   }
 
-  putUpdateTechRecords(
+  createVehicleRecord(newVehicleRecord: VehicleTechRecordModel, user: { id?: string; name: string }) {
+    const recordCopy = cloneDeep(newVehicleRecord);
+
+    const url = `${environment.VTM_API_URI}/vehicles`;
+    const body = {
+      msUserDetails: { msOid: user.id, msUser: user.name },
+      vin: recordCopy.vin,
+      primaryVrm: recordCopy.vrms ? recordCopy.vrms[0].vrm : '',
+      trailerId: recordCopy.trailerId ?? '',
+      techRecord: recordCopy.techRecord
+    };
```

Comment:
would we ever want secondary vrms>

Reply:
Nick says not as part of this user flow. (TFFT)